### DISCUSSION TITLE
How to draw a line with rounded endings in draw.io?

### DISCUSSION THREAD

**Discussion by BlkPingu** [jgraph/drawio#4224] [General]
* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Is your feature request related to a problem? Please describe.**
Not related to a problem, other than not being able to make the shape I need'

**Describe the solution you'd like**
Add the ability to round off endings of lines

**Describe alternatives you've considered**
I have considered loading the svg export into commercial software end post processing my diagram, but that's not ideal.

![EfXA8](https://github.com/jgraph/drawio-desktop/assets/28692167/158f257e-90ea-4dcf-bfc9-12c914e5bd45)

**Comment by Peter-Herrmann** [comment]
1. Select the line you wish to have rounded endings
2. Open the **Style** tab on the **Format** Pane
3. Open the **Properties** dropdown
4. Find the **Line Cap** setting
5. Choose **Round**

Hope that helps :)
